apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/QueryLoggingListener.java

```diff
@@ -192,6 +192,10 @@ static enum QueryType {
             s.contains(" from beneficiary_monthly ")
                 && s.contains("year_month=")
                 && s.contains("partd_contract_number_id="))),
+    /** Dialect-specific verification denoting support for sequence generation. */
+    HIBERNATE_INFORMATION_SCHEMA_SEQUENCES(
+        "hibernate_information_schema_sequences",
+        (s -> s.contains(" from information_schema.sequences"))),
```

Comment:
For future maintainers, it might be helpful to improve the comment a bit to mention this is a query hibernate runs automatically for us in the background, and any other useful info from what was discussed earlier today.

Reply:
I don't disagree. I'll spend a line or two more to assist those in the future.